doc/source/reference/arrays.interface.rst

```diff
@@ -315,25 +316,35 @@ largely aesthetic.  In particular:
 1. The PyArrayInterface structure had no descr member at the end
    (and therefore no flag ARR_HAS_DESCR)
 
-2. The desc member of the PyCObject returned from __array_struct__ was
+2. The ``context`` member of the :c:type:`PyCapsule` (formally the ``desc``
+   member of the ``PyCObject``) returned from ``__array_struct__`` was
    not specified.  Usually, it was the object exposing the array (so
    that a reference to it could be kept and destroyed when the
-   C-object was destroyed).  Now it must be a tuple whose first
-   element is a string with "PyArrayInterface Version #" and whose
-   second element is the object exposing the array.
+   C-object was destroyed). It is now an explicit requirement that this field
+   be used in some way to hold a reference to the owning object.
 
-3. The tuple returned from __array_interface__['data'] used to be a
+   .. note::
+
+       Until August 2020, this said:
+
+           Now it must be a tuple whose first element is a string with "PyArrayInterface Version #" and whose
+           second element is the object exposing the array.
+
+       This design was retracted in <https://mail.python.org/pipermail/numpy-discussion/2006-June/020995.html>, and is an error to have relied
+       upon in the intermediate 14 years.
```

Comment:
"in" <- "it"

Reply:
Should also break the line.

Reply:
Maybe "and its continued use has been an error since that decision."

Reply:
I think that decision was almost immediately after this document was published. Happy to reword this however

Reply:
> Happy to reword this however

I found it a bit confusing, so a rewrite would help.

Reply:
Committed my suggestion below, let me know if that's any better.